package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-backoffice",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
In fact, that will be the version 2.0.0: https://github.com/kuzzleio/kuzzle-backoffice/milestone/4?closed=1